### PR TITLE
fix: set site screen background colors

### PR DESCRIPTION
**Pull request by paulschreiber**
## Description
Set site screen background colors.

Also clean up project screen background color parameters to have consistent formatting.

### Related Issues
Addresses https://github.com/techmatters/terraso-mobile-client/issues/1601

### CODE REVIEW

File: dev-client/src/util.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ export const formatPercent = (value: number) => {
 };
 
 export const formatCoordinate = (value: number) => {
-  return value.toLocaleString(undefined, {
+  return value?.toLocaleString(undefined, {
```
Comment (tm-ruxandra) [inline dev-client/src/util.ts:66]: What does this change (and the one above it in `StaticMapView`)  have to do with site screen colors? I think it should either be explained in the PR or moved to a different PR.

Comment (paulschreiber) [reply dev-client/src/util.ts:66]: It's unrelated. I hit this error during debugging and didn't think it was worth a separate PR. I'll move it.

**Review by tm-ruxandra — APPROVED** [review]
